src/store/helpers.ts

```diff
@@ -280,3 +282,37 @@ export const getCollisions = (nodes: Node[], ui: UIMetaData, nodeSpacing: number
 
     return updatedNodes;
 };
+
+export const getGhostNode = (fromNode: Node, fromNodeUI: UINode, definition: FlowDefinition) => {
```

Comment:
added a helper for `ghostNode` composition so that we can generate ghost nodes in tests